vagrant/src/test/java/org/jclouds/vagrant/compute/VagrantComputeServiceAdapterLiveTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.vagrant.compute;
+
+import java.io.File;
+import java.util.Properties;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.jclouds.compute.domain.NodeMetadata;
+import org.jclouds.compute.domain.TemplateBuilder;
+import org.jclouds.compute.internal.BaseComputeServiceLiveTest;
+import org.jclouds.sshj.config.SshjSshClientModule;
+import org.jclouds.vagrant.reference.VagrantConstants;
+import org.testng.annotations.Test;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.google.inject.Module;
+
+public abstract class VagrantComputeServiceAdapterLiveTest extends BaseComputeServiceLiveTest {
+
+   public VagrantComputeServiceAdapterLiveTest() {
+      provider = "vagrant";
+   }
+
+   @Override
+   protected Module getSshModule() {
+      return new SshjSshClientModule();
+   }
+
+   @Override
+   protected TemplateBuilder templateBuilder() {
+      return super.templateBuilder()
+         .imageId(getImageId());
+   }
```

Comment:
I've just seen this. Instead of hardcoding the fact that you want to use a custom image to run the tests, use the existing "jclouds.template" property.

You can configure a default value in the pom.xml (like in [digitalocean2](https://github.com/jclouds/jclouds/blob/master/providers/digitalocean2/pom.xml#L151) for example), and run the tests with `-Dtest.vagrant.template="imageId=foo/bar"`. This will keep the tests aligned with the rest of the providers.